src/test/java/uk/gov/hmcts/reform/cwrdapi/controllers/response/StaffRefDataJobTitleResponseTest.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.cwrdapi.controllers.response;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+class StaffRefDataJobTitleResponseTest {
+
+    private final StaffRefJobTitleResponse staffRefJobTitleResponse = new StaffRefJobTitleResponse();
+
+    @Test
+    void test_has_mandatory_fields_specified_not_null() {
```

Comment:
add test cases for each missing mandatory fields and add assertions for expected exceptions and error messages.

Reply:
@rajkatla-hmcts Added it